rust-sdk/core/Cargo.toml

```diff
@@ -18,7 +18,7 @@ floats = ["dep:libm"]
 
 [dependencies]
 ethnum = { version = "^1.1" }
-libm = { version = "^0.1", optional = true }
+libm = { version = "^0.2", optional = true }
```

Comment:
quick Q: Why we requre 0.2 ? (old is 0.1).
I don't think 0.2 is a bad idea, but I am concerned that the requirement has been raised here while other areas have been relaxed.

Reply:
Yeah this is the same issue as with the anchor dep. 0.1 and 0.2 are not semver compatible (even though both would work with the library). I should be able to change it to `>=0.1, <0.3` which might be better